template/page-docs.html.jinja

```diff
@@ -1,126 +1,426 @@
+        "title": "Governance and the Amendment Process",
+        "url": "https://www.youtube.com/embed/4GbRdanHoR4?rel=0&amp;showinfo=0&amp;autoplay=1",
+      },
+      {
+        "src": "./assets/img/backgrounds/docs-sidechains@2x.png",
+        "title": "Federated Sidechains",
+        "url": "https://www.youtube.com/embed/NhH4LM8NxgY?rel=0&amp;showinfo=0&amp;autoplay=1",
+      },
+    ]
+  %}
+  {% set useCases = [
+    {
+      "title": "On-Chain Finance",
+      "id": "on-chain-finance-use-cases",
+      "imgClass": "wallet-illustration",
+      "subItems": [
+        {
+          "description": "Trade on the decentralized exchange",
+          "link": "./trade-in-the-decentralized-exchange.html",
+        },
+        {
+          "description": "Make payments",
+          "link": "./send-xrp.html",
+        },
+        {
+          "description": "Use specialized payment types",
+          "link": "./use-specialized-payment-types.html"
+        }
+      ],
+    },
+    {
+      "title": "Tokens",
+      "id": "token-use-cases",
+      "imgClass": "token-illustration",
+      "subItems": [
+        {
+          "description": "Non-fungible Tokens",
+          "link": "./non-fungible-tokens.html",
+        },
+        {
+          "description": "Issue a stablecoin",
+          "link": "./issue-a-fungible-token.html",
+        },
+        {
+          "description": "Assign an authorized minter",
+          "link": "./authorize-minter.html",
+        },
+      ],
+    },
+    {
+      "title": "Payments",
+      "id": "payments-use-cases",
+      "imgClass": "connections-illustration",
+      "subItems": [
+        {
+          "description": "Peer to peer payments",
+          "link": "./direct-xrp-payments.html",
+        },
+        {
+          "description": "Cross-currency payments",
+          "link": "./cross-currency-payments.html",
+        },
+        {
+          "description": "Escrows",
+          "link": "./escrow.html",
+        },
+      ],
+    },
+  ]
+  %}
+  {%
+    set getStartedVideos = [
+    {
+      "src": "./assets/img/backgrounds/docs-intro-to-XRP-ledger@2x.png",
+      "title": "Intro to XRP Ledger",
+      "url": "https://www.youtube.com/embed/sVTybJ3cNyo?rel=0&amp;showinfo=0&amp;autoplay=1",
+    },
+    {
+      "src": "./assets/img/backgrounds/docs-accounts@2x.png",
+      "title": "Accounts",
+      "url": "https://www.youtube.com/embed/eO8jE6PftX8?rel=0&amp;showinfo=0&amp;autoplay=1",
+    },
+    {
+      "src": "./assets/img/backgrounds/docs-decentralized-exchange@2x.png",
+      "title": "Decentralized Exchange",
+      "url": "https://www.youtube.com/embed/VWNrHBDfXvA?rel=0&amp;showinfo=0&amp;autoplay=1",
+    },
+    {
+      "src": "./assets/img/backgrounds/docs-tokenization@2x.png",
+      "title": "Tokenization",
+      "url": "https://www.youtube.com/embed/Oj4cWOiWf4A?rel=0&amp;showinfo=0&amp;autoplay=1",
+    },
+    ]
+  %}
+  {%
+    set devTools = [
```

Comment:
I started writing a long thing about how you could dynamically set this in a more translation-friendly way using `pages|selectattr(...)`, but (a) it's very Dactyl-specific, and (b) the dev tools don't actually have blurbs in the config file where you could the (language-appropriate) one, so you'd have to fix that while you're at it.

So, I guess, nevermind. This is good enough.